index.html

```diff
@@ -1,164 +1,197 @@
-    <img class="circle-1" src="./images/circle-1.svg" alt="background image 1 desktop" />
-    <div class="heading-section-header">
-    <h1 class="header-section-heading" id="About">John mutisya</h1>
-    <h2 class="header-section-heading-2">Software Developer</h2>
-  </div>
-    <p class="header-section-paragraph">I can help you build a product, <br> 
-        feature or website Look through<br>
-          some of my work and experience!
-        If<br>
-          you like what you see and have a <br>
-            project you need coded,  don't<br>
+        <section class="header-section-wrapper">
+          <img
+            class="eclipse-1"
+            src="./images/Ellipse_18.png"
+            alt="background image 1"
+          />
+          <img
+            class="circle-1"
+            src="./images/circle-1.svg"
+            alt="background image 1 desktop"
+          />
+          <div class="heading-section-header">
+            <h1 class="header-section-heading" id="About">John mutisya</h1>
+            <h2 class="header-section-heading-2">Software Developer</h2>
+          </div>
+          <p class="header-section-paragraph">
+            I can help you build a product, <br />
+            feature or website Look through<br />
+            some of my work and experience! If<br />
+            you like what you see and have a <br />
+            project you need coded, don't<br />
             hestiate to contact me.
-    </p>
-  <!-- paragraph - desktop version -->
-  <!-- <section class="desktop-mid-section"> -->
-    <p class="header-section-desktop-paragraph">I can help you build a product, 
-      feature or website  <br>
-      Look through some of my work and experience! If<br>
-        you like what you see and have a project you need <br>
-           coded,  don't hestiate to contact me.
-    </p>
+          </p>
+          <!-- paragraph - desktop version -->
+          <!-- <section class="desktop-mid-section"> -->
+          <p class="header-section-desktop-paragraph">
+            I can help you build a product, feature or website <br />
+            Look through some of my work and experience! If<br />
+            you like what you see and have a project you need <br />
+            coded, don't hestiate to contact me.
+          </p>
 
-    <div class="devider"></div>
-    <div class="circle-2">
-      <!-- <img  src="./images/circle-2.svg" alt="background circle 2 desktop" /> -->
-    </div>
-  </section>
-    <!-- Middle Section -->
-
-    <section id="frame_container">
-      <article class="frame">
-        <section class="languages">
-          <h3>Languages</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
+          <div class="devider"></div>
+          <div class="circle-2">
+            <!-- <img  src="./images/circle-2.svg" alt="background circle 2 desktop" /> -->
```

Comment:
- [ ] To make clean code kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20HTML%20%26%20CSS.)